helium/resource.py

```diff
@@ -51,6 +51,10 @@ def __getattr__(self, attribute):
         and promote it into the attributes of this resource.
 
         """
+
+        if attribute.startswith('__'):
```

Comment:
I'm not sure this is the right way to do this? Why not call super first and deal with failure to respond?